package.json

```diff
@@ -4,24 +4,26 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "scripts": {
-    "storybook": "start-storybook -p 9002 -c .storybook",
-    "test": "jest",
-    "test-coverage": "jest --coverage && scripts/print-coverage-link.sh",
-    "tdd": "jest --watch",
-    "package-term": "yarn workspace @gravitational/teleterm package",
-    "build-term": "yarn workspace @gravitational/teleterm build",
     "build-and-package-term": "yarn build-term && yarn package-term",
-    "start-term": "yarn workspace @gravitational/teleterm start",
-    "start-teleport": "yarn workspace @gravitational/teleport start",
-    "start-teleport-e": "yarn workspace @gravitational/teleport.e start",
+    "build-e": "yarn build-teleport-e",
+    "build-oss": "yarn build-teleport-oss",
     "build-teleport": "yarn build-teleport-oss && yarn build-teleport-e",
-    "build-teleport-oss": "yarn workspace @gravitational/teleport build --output-path=../../dist/teleport/app",
     "build-teleport-e": "yarn workspace @gravitational/teleport.e build --output-path=../../../dist/e/teleport/app",
-    "build-oss": "yarn build-teleport-oss",
-    "build-e": "yarn build-teleport-e",
-    "type-check-and-lint": "yarn type-check && yarn lint",
+    "build-teleport-oss": "yarn workspace @gravitational/teleport build --output-path=../../dist/teleport/app",
+    "build-term": "yarn workspace @gravitational/teleterm build",
     "lint": "eslint --ext .js,.jsx,.ts,.tsx packages/",
-    "type-check": "tsc --noEmit"
+    "package-term": "yarn workspace @gravitational/teleterm package",
+    "prettier-check": "yarn prettier --check ./packages",
+    "prettier-write": "yarn prettier --write ./packages",
```

Comment:
maybe i'm overthinking it, but does the `write` command apply prettier on every file in `./packages`? b/c if it does, i don't think this repo is ready for this command yet. maybe after we go through a review process of prettifying every file (ensuring no intended changes went in), then we can introduce this and a checker for the CI (or could we have something like prettifying `modified` files only?)